### DISCUSSION TITLE
Is runtests.py still needed?

### DISCUSSION THREAD

**Discussion by diazona** [pytest-dev/pytest-localserver#14] [Q&A] [answered]
@redtoad While grepping for `hg` I just noticed the big blob of compressed code in `runtests.py`. Do we still need it? It doesn't seem to be used as part of the current test pipeline, and I'd say it's better to remove it (probably in 0.6) if it's no longer needed.

Also, this is more for my curiosity, but what is it supposed to do? I did manage to decompress it and convert it to Python code, but I couldn't really get a sense of what its purpose is. It _is_ a pretty neat example of a custom importer though :smiley:

**Comment by redtoad** [comment, marked as answer]
We can delete that. At the time (pytest-2.3.5!) that was a recommended way.
